libs/perun/pipes/src/lib/group-members-action-button-disabled-tooltip.pipe.ts

```diff
@@ -7,7 +7,10 @@ import { Urns } from '@perun-web-apps/perun/urns';
   name: 'groupMembersActionButtonDisabledTooltip',
 })
 export class GroupMembersActionButtonDisabledTooltipPipe implements PipeTransform {
-  transform(group: RichGroup, button?: 'add'): string {
+  transform(group: RichGroup, button?: 'add' | 'copy'): string {
+    if (button === 'copy') {
+      return 'GROUP_DETAIL.MEMBERS.COPY_MEMBERS_DISABLED';
+    }
```

Comment:
It seems to me that the copy action is not affected by the other cases. In that case we could just use a direct translation for the tooltip.